src/Services/AddThisScriptManager.php

```diff
@@ -171,6 +171,8 @@ function setAsync($enabled) {
    *   SRP is lost here.
    */
   private function getJsAddThisConfig() {
+
+    // TODO: find out why $language is returning null
```

Comment:
So, the recommended way to access the language setting is to get a language handler service using dependency injection. The use is actually shown here, https://www.drupal.org/node/2133171

I think you'd actually want to inject `Drupal\Core\Language\LanguageManager` using `language_manager` as the service name or `@language_manager` as the argument to this service. Then you can assign this to `$this->languageManager` and get the language with the method `$this->languageManager->getCurrentLanguage()`